portfolio/src/main/webapp/hobbies.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
```

Comment:
For the entire file: Indent should be 2 spaces.

https://google.github.io/styleguide/htmlcssguide.html#Indentation

Reply:
Done!